project_one.html

```diff
@@ -139,15 +222,16 @@ <h2>Stories</h2>
       <br />
 
       <section class="about">
+        <div class="about_side" >
         <h2 class="about_h">About</h2>
         <h2 class="about_h">Myself</h2>
         <p class="para">
-          Hello I’m a software developer! I can help you build a product ,
+          Hello I'm a software developer! I can help you build a product ,
           feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
+          like what you see and have a project you need coded, don't hestiate to
           contact me.
         </p>
```

Comment:
- [ ] In the about section, please make sure the left block takes only 40% of the section width and the right block takes the remaining 60%.
- [ ] Please also make sure there are proper paddings and margins in and around the section so that it matches the original design
- [ ] Kindly also ensure that the width and height of the `Get my resume` button matches the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/160104440-428f7e65-0edb-4892-a326-bde0e858cea7.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/160104555-5ad6b0bc-570c-47c2-aa82-00f3d365c941.png)